docs/openapi/schemas.yaml

```diff
@@ -394,8 +394,32 @@ AuditConfigType:
     disabled:
       description: Whether audits are disabled for the given audit type
       type: boolean
+    excludedUrls:
+      description: (Optional) Set of urls to exclude. If empty, will remove previously excluded URLs. It's an array of strings
+      type: array
+      items:
+        type: string
   example:
     disabled: true
+    excludedUrls:
+      - 'https://www.adobe.com/some-page'
+      - 'https://www.adobe.com/another-page'
+UpdateAuditConfigType:
+  type: object
+  properties:
+    status:
+      description: Set the update operation type
+      type: string
+    excludedUrls:
```

Comment:
Since the update could mean both exclude or re-include (in the future, not now), I'd name the list more generically `URLs` or `targetURLs`.
Otherwise, we could drop the `status` property and just keep the semantics of the update in the name of the property `excludedURLs`.

Reply:
I've removed the `status` property to simplify the implementation.